.github/workflows/build.yml

```diff
@@ -175,6 +177,338 @@ jobs:
         name: aw-android
         path: dist/aw-android*.apk
 
+  test:
```

Comment:
This could be a *lot* faster & shorter if it waited for the build job to complete and retrieve the built `aw-android.apk` that way.